components/Sponsors.jsx

```diff
@@ -0,0 +1,84 @@
+import Image from "next/image";
+import Heading from "./Heading";
+
+const SPONSORS_CONTAINER = "bg-white flex flex-col items-center";

```

Comment:
Can we add `py-20` for some vertical spacing?

Reply:
sounds g